editor/project_export.cpp

```diff
@@ -726,6 +727,8 @@ void ProjectExportDialog::_export_project() {
 }
 
 void ProjectExportDialog::_export_project_to_path(const String &p_path) {
+	// Save this name for use in future exports (but drop the file extension)
+	default_filename = p_path.substr(0, p_path.rfind(".")).get_file();
```

Comment:
You can use http://docs.godotengine.org/en/latest/classes/class_string.html#class-string-get-basename instead of parsing it manually.

Reply:
When fixing that, please amend the commit instead of making a new one, as described in the "Note" box in http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase